src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+import './App.css';
+import BookList from "../containers/BookList";
+import BookForm from "../containers/BookForm";
+
+function App() {
```

Comment:
[Optional]
- Kindly use the ES6 arrow syntax instead of the function keyword :smile: